edx_notifications/server/web/static/edx_notifications/templates/notification_pane.html

```diff
@@ -1,6 +1,35 @@
-<h3>Here are your notifications</h3>
 <ul class='edx-notifications-list'>
     <% _.each(user_notifications, function(user_notification){ %>
         <li class='edx-notifications-list-item'><%= user_notification.html %></li>
     <% }); %>
 </ul>
+<div class="container">
+    <div class="content">
+        <h2>Notifications</h2>
+        <div class="actions">
+            <ul class="notifications_list">
+                <li class="active"><a id="unread_notifications" href="#">View unread</a></li>
+                <li><a id="user_notifications_all" href="#">View all</a></li>
+                <li><a id="mark_notifications_read" href="#">Mark as read</a></li>
+                <li><a id="hide_pane" href="#">Hide</a></li>
+            </ul>
+        </div>
+        <ul>
+            <% if (typeof user_notifications == 'undefined' || user_notifications.length == 0) { %>
+                <li>
+                    <p class="description">You have no unread notifications.</p>
+                </li>
+            <% } else { %>
+            <% _.each(user_notifications, function(user_notification){ %>
+
+            <li>
+                <h3 class="borderB padB5 uppercase bold marB10">Annoucments</h3>
+                <h3 class="marT0"><a href="">Welcome to McKinsey Academy</a></h3>
```

Comment:
We'll have to make that McKinsey Academy some template variable that is passed in. I'll have to think how to do that...